chart/templates/drupal-deployment.yaml

```diff
@@ -35,11 +35,7 @@ spec:
             port: 9000
         readinessProbe:
           exec:
-            command:
-            - /bin/sh
-            - -c
-            - if [ ! {{ include "drupal.deployment-in-progress-test" . }} ]; then timeout -t 3 drush check-bootstrap; else exit 1; fi
-          periodSeconds: 3
+            command: ['/bin/sh', '-c', 'if [ {{ include "drupal.deployment-in-progress-test" . }} ]; then exit 1; fi']
```

Comment:
To be more verbose I suggest:
`/bin/sh -x -c ...`
Extra log saves time when things go wrong.

Reply:
Good suggestion @ragnarkurmwunder . I believe this issue might be in several places, could you take a look at the code and make a PR for it, please? I'll try to bring it over to https://github.com/wunderio/charts when we test if on k8s

Reply:
Actually, you can simplify it too :)
```
{{ include "drupal.deployment-in-progress-test" . }} && exit 1
```